### ISSUE TITLE
Hypomanic Episode

### ISSUE THREAD

**Issue by SmartMonkey-git** [obophenotype/human-phenotype-ontology#11586]
**Preferred term label:**
Hypomanic Episode

**Synonyms**
Hypomania

**Definition (free text, please give PubMed ID)**
An abnormal, distinct period characterized by noticeable changes in both mood and energy that lasts for a minimum duration of 4 days. It is a less severe form of mania that definitionally does not cause functional impairment.
PMID: [38694998](https://pubmed.ncbi.nlm.nih.gov/38694998/)

**Parent term (use hpo.jax.org/app)**
HP:0100851/Abnormal emotional state and HP:5200263/Abnormally increased volition
Those are the same as mania.

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0009-0002-1118-4784

**Comment by pnrobinson** [comment]
@SmartMonkey-git  I think this is not q good HPO term, it is more of a disease diagnosis. Let's think of a better way to represent things like this!
